### PR TITLE
doc: add local_cluster doc

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @joshix

### CODE REVIEW

File: Documentation/local_cluster.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+$ export ETCDCTL_API=3
+
+$ ./etcdctl put foo bar
+OK
+
+$ ./etcdctl get foo
+bar
+```
+
+## Local Multi-members Cluster
```
Comment (heyitsanthony) [inline Documentation/local_cluster.md]: s/Multi-members/Multi-member/

File: Documentation/local_cluster.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+A Procfile is provided to easily set up a local multi-members cluster. Start a multi-members cluster with a few commands:
```
Comment (heyitsanthony) [inline Documentation/local_cluster.md]: s/Multi-members/Multi-member/

File: Documentation/local_cluster.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+```
+# install goreman program to control Profile-based applications.
+$ go get github.com/mattn/goreman
+$ goreman -f Procfile start
+...
+```
+
+The started members listens on `localhost:12379`, `localhost:22379` and `localhost:32379` for client requests respectively.
```
Comment (heyitsanthony) [inline Documentation/local_cluster.md]: s/listens/listen/
s/ and/, and/

File: Documentation/local_cluster.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+| 8211f1d0f64f3269 | started | infra1 | http://127.0.0.1:12380 | http://127.0.0.1:12379 |
+| 91bc3c398fb3c146 | started | infra2 | http://127.0.0.1:22380 | http://127.0.0.1:22379 |
+| fd422379fda50e48 | started | infra3 | http://127.0.0.1:32380 | http://127.0.0.1:32379 |
++------------------+---------+--------+------------------------+------------------------+
+
+$ etcdctl --endpoints=localhost:12379 put foo bar
+OK
+```
+
+To experience etcd's fault tolerance, kill a member:
```
Comment (heyitsanthony) [inline Documentation/local_cluster.md]: s/experience/exercise/

**Comment by xiang90** [comment]
All fixed. PTAL.

File: Documentation/local_cluster.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+# Setup a Local Cluster
+
+For testing and development deployments, the quickest and easiest way is to setup a local cluster. For a production deployment, refer to [clustering][clustering] section.
```
Comment (heyitsanthony) [inline Documentation/local_cluster.md]: s/refer to/refer to the/

**Comment by heyitsanthony** [comment]
lgtm

File: Documentation/local_cluster.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+# Setup a Local Cluster
+
+For testing and development deployments, the quickest and easiest way is to setup a local cluster. For a production deployment, refer to the [clustering][clustering] section.
```
Comment (joshix) [inline Documentation/local_cluster.md]: s/setup/set up/

**Comment by joshix** [comment]
One exceedingly tiny nit inline; otherwise LGTM

**Comment by xiang90** [comment]
@joshix Thanks!

**Pull request merged**